.github/workflows/Mypy.yml

```diff
@@ -0,0 +1,20 @@
+name: Mypy
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Mypy
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: |
+        pip install mypy
```

Comment:
Can you modify this to install from the requirements.txt folder, and add mypy as a dependency?

(If you've installed mypy into a virtual enviroment, run `pip freeze > requirements.txt` in the backend folder to write mypy and the version number)